docs/core/tools/enable-tab-autocomplete.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Enable tab completion
+description: This article teaches you how to enable tab completion for the .NET Core CLI for PowerShell, Bash, and zsh.
+author: thraka
+ms.author: adegeo
+ms.date: 12/17/2018
+---
+
+# How to enable TAB completion for the .NET Core CLI
+
+Starting with .NET Core SDK v2.0, the .NET Core CLI supports tab completion. This article describes how to configure tab completion for three shells, PowerShell, Bash, and zsh. Other shells may have support for auto completion. Refer to their documentation on how to configure auto completion, the steps should be similar to the steps described in this article.
```

Comment:
Just noticed this 
```suggestion
Starting with .NET Core 2.0 SDK, the .NET Core CLI supports tab completion. This article describes how to configure tab completion for three shells, PowerShell, Bash, and zsh. Other shells may have support for auto completion. Refer to their documentation on how to configure auto completion, the steps should be similar to the steps described in this article.
```

Reply:
Strange, I thought I committed that suggestion